src/contributing/adding_new_operation_how_to.md

```diff
@@ -0,0 +1,108 @@
+# Adding operations to Parsec
+
+This mini how-to will cover how to add operations to the backend and frontend of Parsec. The steps
```

Comment:
"Backend" and "Frontend" have a different meaning, than what they usually use to describe, in Parsec. See the [repository map](https://parallaxsecond.github.io/parsec-book/parsec_service/source_code_structure.html#repository-map) diagram. `Frontend` parts are the listeners + front end handler and "backend" parts are the dispatcher and back end handler. I think to not confuse anybody we should probably use other terms throughout this document.

> This mini how-to will cover how to add operations to the backend and frontend of Parsec.

Can maybe be replaced with: "This mini how-to will cover how to add operations to the Parsec service and Rust client."

We should also maybe put a note that the `parsec-interface-rs` and client parts are only for the Rust client but should be similar for other clients.

Reply:
Isn't `parsec-interface-rs` used to go between Protobuf and Parsec? (so it is always used?)

Reply:
> We should also maybe put a note that the parsec-interface-rs and client parts are only for the Rust client but should be similar for other clients.

I thought about that, the reason I put `parsec-client-rust` as part of the procedure (i.e. not marked as optional) is the fact that Parsec's e2e tests require it, so it has to be done in order for Parsec to pass CI.

Reply:
Yeah both of those points are correct and you are right 😄